<ui version="4.0" >
 <class>qmwMainWindow</class>
 <widget class="QMainWindow" name="qmwMainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>File Commander</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <property name="spacing" >
     <number>6</number>
    </property>
    <property name="leftMargin" >
     <number>9</number>
    </property>
    <property name="topMargin" >
     <number>9</number>
    </property>
    <property name="rightMargin" >
     <number>9</number>
    </property>
    <property name="bottomMargin" >
     <number>9</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget" >
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="leftMargin" >
           <number>0</number>
          </property>
          <property name="topMargin" >
           <number>0</number>
          </property>
          <property name="rightMargin" >
           <number>0</number>
          </property>
          <property name="bottomMargin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QComboBox" name="qcbLeftDrive" >
            <property name="maximumSize" >
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="focusPolicy" >
             <enum>Qt::NoFocus</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="qlLeftDriveInfo" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>TextLabel</string>
            </property>
            <property name="wordWrap" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="qpbLeftRootDir" >
            <property name="maximumSize" >
             <size>
              <width>20</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="focusPolicy" >
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text" >
             <string>\</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="qpbLeftUpDir" >
            <property name="maximumSize" >
             <size>
              <width>20</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="focusPolicy" >
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text" >
             <string>..</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="leftMargin" >
           <number>0</number>
          </property>
          <property name="topMargin" >
           <number>0</number>
          </property>
          <property name="rightMargin" >
           <number>0</number>
          </property>
          <property name="bottomMargin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="qleLeftPath" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="palette" >
             <palette>
              <active>
               <colorrole role="Base" >
                <brush brushstyle="SolidPattern" >
                 <color alpha="255" >
                  <red>236</red>
                  <green>233</green>
                  <blue>216</blue>
                 </color>
                </brush>
               </colorrole>
              </active>
              <inactive>
               <colorrole role="Base" >
                <brush brushstyle="SolidPattern" >
                 <color alpha="255" >
                  <red>236</red>
                  <green>233</green>
                  <blue>216</blue>
                 </color>
                </brush>
               </colorrole>
              </inactive>
              <disabled>
               <colorrole role="Base" >
                <brush brushstyle="SolidPattern" >
                 <color alpha="255" >
                  <red>236</red>
                  <green>233</green>
                  <blue>216</blue>
                 </color>
                </brush>
               </colorrole>
              </disabled>
             </palette>
            </property>
            <property name="readOnly" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="qpbLeftFavourites" >
            <property name="maximumSize" >
             <size>
              <width>20</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="focusPolicy" >
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text" >
             <string>*</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="qpbLeftHistory" >
            <property name="maximumSize" >
             <size>
              <width>20</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="focusPolicy" >
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text" >
             <string>PushButton</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QStackedWidget" name="qswLeft" >
          <property name="currentIndex" >
           <number>0</number>
          </property>
          <widget class="QWidget" name="page" />
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="leftMargin" >
           <number>0</number>
          </property>
          <property name="topMargin" >
           <number>0</number>
          </property>
          <property name="rightMargin" >
           <number>0</number>
          </property>
          <property name="bottomMargin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="qlLeftSelected" >
            <property name="text" >
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="qleLeftQuickSearch" />
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget" >
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="leftMargin" >
           <number>0</number>
          </property>
          <property name="topMargin" >
           <number>0</number>
          </property>
          <property name="rightMargin" >
           <number>0</number>
          </property>
          <property name="bottomMargin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QComboBox" name="qcbRightDrive" >
            <property name="maximumSize" >
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="focusPolicy" >
             <enum>Qt::NoFocus</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="qlRightDriveInfo" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>TextLabel</string>
            </property>
            <property name="wordWrap" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="qpbRightRootDir" >
            <property name="maximumSize" >
             <size>
              <width>20</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="focusPolicy" >
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text" >
             <string>\</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="qpbRightUpDir" >
            <property name="maximumSize" >
             <size>
              <width>20</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="focusPolicy" >
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text" >
             <string>..</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="leftMargin" >
           <number>0</number>
          </property>
          <property name="topMargin" >
           <number>0</number>
          </property>
          <property name="rightMargin" >
           <number>0</number>
          </property>
          <property name="bottomMargin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="qleRightPath" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="palette" >
             <palette>
              <active>
               <colorrole role="Base" >
                <brush brushstyle="SolidPattern" >
                 <color alpha="255" >
                  <red>236</red>
                  <green>233</green>
                  <blue>216</blue>
                 </color>
                </brush>
               </colorrole>
              </active>
              <inactive>
               <colorrole role="Base" >
                <brush brushstyle="SolidPattern" >
                 <color alpha="255" >
                  <red>236</red>
                  <green>233</green>
                  <blue>216</blue>
                 </color>
                </brush>
               </colorrole>
              </inactive>
              <disabled>
               <colorrole role="Base" >
                <brush brushstyle="SolidPattern" >
                 <color alpha="255" >
                  <red>236</red>
                  <green>233</green>
                  <blue>216</blue>
                 </color>
                </brush>
               </colorrole>
              </disabled>
             </palette>
            </property>
            <property name="readOnly" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="qpbRightFavourites" >
            <property name="maximumSize" >
             <size>
              <width>20</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="focusPolicy" >
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text" >
             <string>*</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="qpbRightHistory" >
            <property name="maximumSize" >
             <size>
              <width>20</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="focusPolicy" >
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text" >
             <string>PushButton</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QStackedWidget" name="qswRight" >
          <property name="currentIndex" >
           <number>0</number>
          </property>
          <widget class="QWidget" name="page_3" />
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="leftMargin" >
           <number>0</number>
          </property>
          <property name="topMargin" >
           <number>0</number>
          </property>
          <property name="rightMargin" >
           <number>0</number>
          </property>
          <property name="bottomMargin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="qlRightSelected" >
            <property name="text" >
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="qleRightQuickSearch" />
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="qlGlobalPath" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize" >
         <size>
          <width>16777215</width>
          <height>16</height>
         </size>
        </property>
        <property name="text" >
         <string>TextLabel</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>></string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="qcbCommand" >
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="editable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="qpbTerminal" >
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text" >
         <string>Terminal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="qpbView" >
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text" >
         <string>View</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="qpbEdit" >
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text" >
         <string>Edit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="qpbCopy" >
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text" >
         <string>Copy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="qpbMove" >
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text" >
         <string>Move</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="qpbNewDirectory" >
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text" >
         <string>New directory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="qpbDelete" >
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text" >
         <string>Delete</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="qaUnpackSelectedFiles" />
    <addaction name="separator" />
    <addaction name="qaQuit" />
   </widget>
   <widget class="QMenu" name="menu_Mark" >
    <property name="title" >
     <string>&amp;Mark</string>
    </property>
    <addaction name="qaSelectGroup" />
    <addaction name="qaUnselectGroup" />
    <addaction name="qaSelectAll" />
    <addaction name="qaUnselectAll" />
    <addaction name="qaInvertSelection" />
    <addaction name="separator" />
    <addaction name="qaCompareDirectories" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="qaAbout" />
   </widget>
   <widget class="QMenu" name="menuCommands" >
    <property name="title" >
     <string>&amp;Commands</string>
    </property>
    <addaction name="qaSearch" />
    <addaction name="qaFavouriteDirectories" />
    <addaction name="qaHistoryDirectoryList" />
    <addaction name="separator" />
    <addaction name="qaBranchView" />
    <addaction name="separator" />
    <addaction name="qaFullScreen" />
   </widget>
   <widget class="QMenu" name="qmPanel" >
    <property name="title" >
     <string>&amp;Panel</string>
    </property>
    <addaction name="qaColumnSet" />
    <addaction name="separator" />
    <addaction name="qaColumnsSeparator" />
    <addaction name="separator" />
    <addaction name="qaReverseOrder" />
    <addaction name="separator" />
    <addaction name="qaRefresh" />
   </widget>
   <widget class="QMenu" name="menuConfiguration" >
    <property name="title" >
     <string>C&amp;onfiguration</string>
    </property>
    <addaction name="qaOptions" />
    <addaction name="qaSavePosition" />
    <addaction name="qaSaveSettings" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menu_Mark" />
   <addaction name="menuCommands" />
   <addaction name="qmPanel" />
   <addaction name="menuConfiguration" />
   <addaction name="menuHelp" />
  </widget>
  <action name="qaQuit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="qaOptions" >
   <property name="text" >
    <string>&amp;Options...</string>
   </property>
  </action>
  <action name="qaSavePosition" >
   <property name="text" >
    <string>&amp;Save position</string>
   </property>
  </action>
  <action name="qaSearch" >
   <property name="text" >
    <string>&amp;Search...</string>
   </property>
  </action>
  <action name="qaRefresh" >
   <property name="text" >
    <string>&amp;Refresh</string>
   </property>
  </action>
  <action name="qaFullScreen" >
   <property name="text" >
    <string>&amp;Full screen</string>
   </property>
  </action>
  <action name="qaSelectGroup" >
   <property name="text" >
    <string>&amp;Select group...</string>
   </property>
  </action>
  <action name="qaUnselectGroup" >
   <property name="text" >
    <string>&amp;Unselect group...</string>
   </property>
  </action>
  <action name="qaSelectAll" >
   <property name="text" >
    <string>Select &amp;all</string>
   </property>
  </action>
  <action name="qaUnselectAll" >
   <property name="text" >
    <string>U&amp;nselect all</string>
   </property>
  </action>
  <action name="qaInvertSelection" >
   <property name="text" >
    <string>Invert s&amp;election</string>
   </property>
  </action>
  <action name="qaSaveSettings" >
   <property name="text" >
    <string>Save &amp;settings</string>
   </property>
  </action>
  <action name="qaCompareDirectories" >
   <property name="text" >
    <string>&amp;Compare directories</string>
   </property>
  </action>
  <action name="qaBranchView" >
   <property name="text" >
    <string>&amp;Branch view</string>
   </property>
  </action>
  <action name="qaColumnsSeparator" >
   <property name="text" >
    <string>ColumnsSeparator</string>
   </property>
   <property name="visible" >
    <bool>false</bool>
   </property>
  </action>
  <action name="qaReverseOrder" >
   <property name="text" >
    <string>Reverse &amp;order</string>
   </property>
  </action>
  <action name="qaAbout" >
   <property name="text" >
    <string>&amp;About</string>
   </property>
  </action>
  <action name="qaFavouriteDirectories" >
   <property name="text" >
    <string>Favourite &amp;directories</string>
   </property>
  </action>
  <action name="qaColumnSet" >
   <property name="text" >
    <string>&amp;Column set</string>
   </property>
  </action>
  <action name="qaUnpackSelectedFiles" >
   <property name="text" >
    <string>&amp;Unpack selected files...</string>
   </property>
  </action>
  <action name="qaHistoryDirectoryList" >
   <property name="text" >
    <string>&amp;History directory list</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>qaQuit</sender>
   <signal>triggered()</signal>
   <receiver>qmwMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>296</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
